sdk/iot/azure-iot-modelsrepository/samples/README.md

```diff
@@ -0,0 +1,12 @@
+# Azure IoT Models Repository Library Samples
+
+This directory contains samples showing how to use the features of the Azure IoT Models Repository Library.
+
+The pre-configured endpoints and DTMIs within the sampmles refer to example DTDL documents that can be found on [devicemodels.azure.com](https://devicemodels.azure.com/). These values can be replaced to reflect the locations of your own DTDLs, wherever they may be.
```

Comment:
spelling `samples`

Reply:
"These values can be replaced to reflect the locations of your own **DTDL models**, wherever they may be."